modules/API/pages/built-in-endpoints.adoc

```diff
@@ -739,148 +739,227 @@ $ curl -X GET 'http://localhost:9000/deleted_vertex_check?threadnum=10&verbose=0
+==== Request body
+The endpoint expects a JSON request body in the following format:
 
-==== Sample request:
+[,javascript]
+----
+{
+    "secret": <secret_value>, <1>
+    "graph": <graph_name>, <2>
+    "lifetime": <token_lifetime> <3>
+}
+----
+<1> User's secret to generate the token.
+Required if the request body does not supply `graph`.
+<2> Name of the graph that the token is valid for.
+Required if the request body does not supply `secret`.
+<3> Period of time for which the token is valid measured in seconds. The default value is about 2.6 million (about a month).
 
-[source,bash]
+==== Sample request
+[tabs]
+====
+With secret::
++
+--
+[source,console]
+----
+$ curl -X PUT http://localhost:9000/requesttoken \
+     -d '{"secret": "jiokmfqqfu2f95qs6ug85o89rpkneib3", "token": "tohvf6khjqju8jf0r0l1cohhlm8gi5fq", "lifetime": "15000"}'
+----
+--
+With username-password pair::
++
+--
+[source,console]
+----
+$ curl --user example_username:example_password \
+     -X PUT http://localhost:9000/requesttoken \
+     -d '{"token": "tohvf6khjqju8jf0r0l1cohhlm8gi5fq", "lifetime": "15000"}'
+----
+--
+Response::
++
+--
+[source.wrap,json]
 ----
-curl -d <path_to_secret> -X POST \
-"http://localhost:9000/requesttoken?lifetime=1000000"
 {
   "code": "REST-0000",
-  "expiration": 0,
+  "expiration": 1641554169,
   "error": false,
   "message": "Refresh token successfully.",
-  "token": "tohvf6khjqju8jf0r0l1cohhlm8gi5fq"
+  "token": "0mq98l9pderkaivndf820gudg923p3l0"
 }
 ----
+--
+====
 
-Replace `path_to_secret` with the path to the file containing your secret. The file should only include a single line, which is your secret.
-
-==== Parameters:
 
-|===
-| Name | Required | Description
+==== Parameters
 
-| `lifetime`
-| No
-| Period of time for which the token is valid measured in seconds. The default value is about 2.6 million (about a month).
-|===
+None.
 
-=== Refresh a token
 
-`PUT /requesttoken`
-
-This endpoint takes a token and its associated secret and refreshes the lifetime of the token. The token itself remains unchanged.
+=== Delete a token
 
-*Parameters:*
+`DELETE /requesttoken`
 
-|===
-| *Name* | Required | Description
+A user can delete a token using either
 
-| `token`
-| Yes
-| Token to refresh.
+* A secret, which is associated with the token to be deleted
+* Their username and password in their request header
 
-| `secret`
-| Yes
-| User's secret used to generate the token.
+==== Request body
+The endpoint expects a JSON request body in the following format:
 
-| `lifetime`
-| Yes
-| Period of time for which the token is valid measured in seconds.
-|===
+[,javascript]
+----
+{
+    "secret": <secret_value>, <1>
+    "graph": <graph_name>, <2>
+}
+----
+<1> User's secret to generate the token.
+Required if the request body does not supply `graph`.
+<2> Name of the graph that the token is valid for.
+Required if the request body does not supply `secret`.
```

Comment:
This part doesn't seem to be consistent with the example below. Please update accordingly.